src/VisualStudio/Razor/project.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "dependencies": {
+    },
+    "frameworks": {
+        "net46": { }
```

Comment:
Runtime is only required for executable code where the dependencies will be required to exist in the same folder as the running program (exe and unit test libraries for example).

Libraries do not need a runtime declaration.